.yarn/patches/react-scripts-npm-5.0.1-d06bd2d5ad.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/config/webpack.config.js b/config/webpack.config.js
+index e465d8e7a00ec889e9c4c29ba7812019f8319a41..24faad3f65c1cf117b749a292493332cc0b68a48 100644
+--- a/config/webpack.config.js
++++ b/config/webpack.config.js
+@@ -345,6 +345,12 @@ module.exports = function (webpackEnv) {
+           babelRuntimeRegenerator,
+         ]),
+       ],
++      fallback: {
++        assert: require.resolve("assert-browserify"),
++        buffer: require.resolve("buffer-browserify"),
++        crypto: require.resolve("crypto-browserify"),
++        stream: require.resolve("stream-browserify"),
++      },
+     },
```

Comment:
Some dependencies, such as sjcl, use Node built-in libraries, which isn't possible in the browser.

webpack 5 *no longer provides automatic polyfills*, so this is required in order to use those dependencies.